backend.native/tests/runtime/workers/worker10.kt

```diff
@@ -163,4 +163,22 @@ val stableHolder2 = StableRef.create(("hello" to "world").freeze())
     semaphore.increment()
     future.result
     worker.requestTermination().result
+}
+
+@Test fun runTest6() {
```

Comment:
This test doesn't fail even without the fix applied.

Reply:
Yeah, better use fresh atomic ref here.